assets/react/v3/entries/course-builder/components/curriculum/question-types/ImageAnswering.tsx

```diff
@@ -161,37 +187,12 @@ const ImageAnswering = () => {
         )}
       </DndContext>
 
-      <button
-        type="button"
-        onClick={() => {
-          appendOption(
-            {
-              _data_status: QuizDataStatus.NEW,
-              is_saved: false,
-              answer_id: nanoid(),
-              answer_title: '',
-              is_correct: '0',
-              belongs_question_id: activeQuestionId,
-              belongs_question_type: 'image_answering',
-              answer_order: optionsFields.length,
-              answer_two_gap_match: '',
-              answer_view_format: '',
-            },
-            {
-              shouldFocus: true,
-              focusName: `questions.${activeQuestionIndex}.question_answers.${optionsFields.length}.answer_title`,
-            },
-          );
-
-          if (validationError?.type === 'add_option') {
-            setValidationError(null);
-          }
-        }}
-        css={styles.addOptionButton}
-      >
-        <SVGIcon name="plus" height={24} width={24} />
-        {__('Add Option', 'tutor')}
-      </button>
+      <div>
```

Comment:
Why not passing the icon as props. In that case no need to write any extra css

Reply:
Even if I pass the icon as props, I still need to use custom styles to match color of text.